elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -215,9 +215,12 @@ def get_updated_advisory_rhsa(logger, cve_boilerplate: dict, advisory: Erratum,
 
     highest_impact = get_highest_security_impact(flaw_bugs)
     if highest_impact != advisory.security_impact:
-        logger.info(f'Adjusting advisory security impact from {advisory.security_impact} to {highest_impact}')
-        advisory.update(security_impact=highest_impact)
-        updated = True
+        if compare_security_impact(advisory.security_impact, highest_impact):
```

Comment:
Why not have 
`if impact[advisory.security_impact] < impact[highest_impact]` in this line?

Reply:
good point, updated